elastio-lambda-stream-backup/app/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# syntax=docker/dockerfile:1
+ARG REGISTRY=public.ecr.aws/elastio-dev/
+
+# Possible values: amd64 or arm64
+ARG ARCH=arm64
+# Version elastio cli what will use like base image
+ARG VERSION_TAG=0.8.0-master
```

Comment:
This isn't a good idea, it's the latest 0.8 build from master.  It will change often and be broken sometimes.  Use a release tag, with a comment that this is probably not the latest version anymore and customers should replace this with the latest shipping version.

Reply:
I will put a similar comment.